catalog.html

```diff
@@ -53,9 +58,32 @@ <h1 class="inner-header-title">Гостиницы Седоны</h1>
         <ul class="breadcrumbs">
           <li class="breadcrumbs-item">
             <a class="breadcrumbs-link" href="index.html">
-              <img class="breadcrumbs-link" src="images/icon/home-for-breadcrumps.svg" width="12" height="12" alt="Вернуться на главную страницу.">
+              <svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg" alt="Вернуться на главную страницу.">
+                <g clip-path="url(#clip0_31961_427)">
+                <path d="M6 0.300003L0 5.8V12H12V5.8L6 0.300003ZM11 11H1V6.2L6 1.7L11 6.2V11Z" fill="white"/>
+                <path d="M4 10H5V7.5H7V10H8V6.5H4V10Z" fill="white"/>
+                </g>
+                <defs>
+                <clipPath id="clip0_31961_427">
+                <rect width="12" height="12" fill="white"/>
```

Comment:
![image](https://github.com/slava560730/sedona-home/assets/52457249/09b30d93-2e5b-4259-a2c6-003ed8c27e77)
Это заголовок который отображается во вкладке в браузере
поэтому можно для главной страницы написать Седона главная
для каталога: Седона каталог
некорректно писать title на английском если у нас в теге html lang=ru
```suggestion
               <html lang="ru">
```